app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -229,14 +213,14 @@ public function recent(Organization $suborganization)
      *
      * Get a list of the default projects.
      *
-     * @urlParam suborganization required The Id of a suborganization Example: 1
```

Comment:
Need to add docs as per following conventions
https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api#specifying-request-parameters

Need "@urlParam suborganization"